src/control/lib/daos/pool_property_test.go

```diff
@@ -207,15 +207,15 @@ func TestControl_PoolProperties(t *testing.T) {
 			value:  "-1",
 			expErr: errors.New("invalid"),
 		},
-		"policy-valid": {
-			name:    "policy",
-			value:   "type=io_size",
-			expStr:  "policy:type=io_size",
-			expJson: []byte(`{"name":"policy","description":"Tier placement policy","value":"type=io_size"}`),
-		},
-		"policy-invalid": {
-			name:   "policy",
-			value:  "deadd00d",
+		"data_thresh-valid": {
+			name:    "data_thresh",
+			value:   "8KiB",
+			expStr:  "data_thresh:8.0 KiB",
+			expJson: []byte(`{"name":"data_thresh","description":"Data bdev threshold size","value":8192}`),
+		},
+		"data_thresh-invalid": {
+			name:   "data_thresh",
+			value:  "-2",
 			expErr: errors.New("invalid"),
 		},
```

Comment:
Should have tests for special case values e.g. 0 and ensure in unit-tests that the special case rather than the normal logic gets activated.

Reply:
Thanks, will fix the two go-related thing in a separate patch.